packages/backend/src/asset/service.test.ts

```diff
@@ -115,6 +116,16 @@ describe('Asset Service', (): void => {
         AssetError.DuplicateAsset
       )
     })
+
+    test('Cannot create asset with scale > 255', async (): Promise<void> => {
+      const options = {
+        code: 'ABC',
+        scale: 256
```

Comment:
You could also test with a negative scale

Reply:
I was sure the `unsigned` constraint would work but I wasn't sure the `check` would so that is why I included this test.